docs/jest-integration.md

```diff
@@ -299,7 +374,7 @@ To generate a coverage report, run jest with the `--coverage` flag:
 npx jest --coverage
 ```
 
-Note that unlike for the Jazzer.js CLI Jest only accepts the long flag of
+Note that unlike the Jazzer.js CLI Jest only accepts the long flag of
```

Comment:
"CLI Jest" -> "CLI, Jest"
We should definitely have equal flags for all!

Reply:
Done.